errors/runtime-css-import.md

```diff
@@ -0,0 +1,63 @@
+# Runtime CSS import
+
+#### Why This Error Occurred
+
+Runtime CSS imports are bad for perfomance because the browser can't know about the `@import` until the stylesheet is downloaded. This means the stylesheets can't be fetched in parallel.
+
+#### Possible Ways to Fix It
+
+Import the CSS file in a JavaScript module or use a relative `@import` in your CSS.
+
+**Before**
+
+```css
+@import url('/styles.css');
```

Comment:
The nuance is that this starts with a `/`, right? Might be worth pointing out

Reply:
Added a clarifying comment above the `@import`.